jetstream/analysis.py

```diff
@@ -642,6 +642,13 @@ def run(self, current_date: datetime, dry_run: bool = False) -> None:
 
     def ensure_enrollments(self, current_date: datetime) -> None:
         """Ensure that enrollment tables for experiment are up-to-date or re-create."""
+        if (
+            hasattr(self.config.experiment, "is_enrollment_paused")
```

Comment:
Does `is_enrollment_paused` get set manually through Experimenter? And does it mean that clients will stop enrolling into the experiment until it is set to false?

Reply:
Basically yes. My understanding is that the `is_enrollment_paused` is `False` until enrollment ends at which point it becomes `True`, and enrollment ending is set manually through Experimenter. I'm not sure if there is a separate field for what is set in Experimenter vs this flag here, but in practice I don't think there is an important distinction. Either way, `is_enrollment_paused == True` means enrollment is ended and clients are no longer enrolling.